extensions/wikia/Flags/Flags.i18n.php

```diff
@@ -28,5 +31,12 @@
 	'flags-edit-modal-close-button-text' => 'Text on the button that closes flags edit modal.',
 	'flags-edit-modal-done-button-text' => 'Text on the button that submits changes done to flags.',
 	'flags-edit-modal-no-flags-on-community' => 'Message on modal appearing when there are no flags types defined on the wiki.',
-	'flags-edit-modal-title' => 'Title of the form for editing flags displayed on headline of modal containing the form.'
+	'flags-edit-modal-title' => 'Title of the form for editing flags displayed on headline of modal containing the form.',
+	'log-name-flags' => 'Name of log type displayed on Special:Log',
```

Comment:
Please, form message keys from the most general to the most detaild e.g. 'flags-log-name'.